source/mongodump.txt

```diff
@@ -9,6 +9,9 @@ mongodump
 .. binary:: mongodump
 
 .. program:: mongodump
+
+.. meta:: 
+  :description: mongodump is a MongoDB database tool that creates a binary export of a database's contents.
```

Comment:
Consider adding a sentence that mentions backups and migration. I think these are important ideas to mention because we'd want to surface mongodump for users searching for these terms.

```suggestion
  :description: mongodump is a MongoDB database tool that creates a binary export of a database's contents. Use mongodump to backup your data or migrate to a new deployment."
```

Reply:
Done!